src/test/config.sol

```diff
@@ -100,7 +98,7 @@ contract Config {
         // Values for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0xa963cc9e4F1Dc187d561830ee299040C582e2BfC),        // populate with deployed spell if deployed
+            deployed_spell:                 address(0),        // populate with deployed spell if deployed
             deployed_spell_created:         1655907493,        // use get-created-timestamp.sh if deployed
             previous_spell:                 address(0),        // supply if there is a need to test prior to its cast() function being called on-chain.
             office_hours_enabled:           false,             // true if officehours is expected to be enabled in the spell
```

Comment:
Going to want to set this to `true`

Reply:
done in https://github.com/makerdao/spells-mainnet/pull/258/commits/1de72744e3027d30d69b974552e1dc93df00ce9c